.vs/cscore/v17/DocumentLayout.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Version": 1,
+  "WorkspaceRootPath": "C:\\Users\\Haack\\source\\repos\\cscore\\",
```

Comment:
Please remove the entire .vs folder from the PR, it even contains a few paths like this one 

You can either delete the folder from your disk or only remove it from the git index 
I checked the root gitignore, it already contains as the first line `.vs/` so I am not sure how this got in here